lib/classes/package_detail_builder.rb

```diff
@@ -0,0 +1,14 @@
+class PackageDetailBuilder
```

Comment:
Whilst it's great to see a builder class, I'd like to see if possible to remove this by getting accepts_nested_attributes_for working